### DISCUSSION TITLE
The user agent header does not arrive correctly

### DISCUSSION THREAD

**Discussion by devupmaster** [nextauthjs/next-auth#3094] [Help]
### Question 💬

I'm using a request package inside the credential provider, but in my API the user agent header doesn't arrive correctly. And that only happens with the request inside the next-auth provider. Any other requests outside of next-auth are sent correctly.

If anyone can give me any tips on how to fix this.

### How to reproduce ☕️

provider code:

```
Providers.Credentials({
  name: 'Credentials',
  authorize: async ({ email, password }) => {

    const variables = { email, password }

    try {

      const { data } = await graphQLClient.request({ query: LOGIN, variables })

      if (data.auth.user)
        return data.auth

    } catch (e) {

      throw '/login?error=true'
    }
  }
})
```

how is it coming:

![image](https://user-images.githubusercontent.com/92178071/139771704-4efc39e3-f63d-47b1-9d1c-fcc208539735.png)

how it should be:

![image](https://user-images.githubusercontent.com/92178071/139771843-2e561cd3-f16d-4f13-804b-fc0a36b2a019.png)

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
You initiate the request from the backend, why do you expect that it will have the user's user agent?
